tests/test-recipes/metadata/_clean_rpaths/meta.yaml

```diff
@@ -0,0 +1,27 @@
+package:
+  name: clean_rpaths
+  version: 0.0.1
+
+build:
+  number: 0
+  skip: True  # [not osx]
+  script:
+    - set -x
+    - export
+    - echo "int main(){}" > hello.c
+    - ${CC} hello.c -Wl,-rpath,${PREFIX}/lib -Wl,-rpath,${PREFIX}/lib -Wl,-rpath,${BUILD_PREFIX}/lib -Wl,-rpath,${SRC_DIR} -o hello
```

Comment:
In the last CI run the test failed with
```
/private/tmp/cb/popen-gw2/test_clean_rpaths0/clean_rpaths_1632926362791/work/conda_build.sh: line 5: hello.c: command not found
```
as if `${CC}` wasn't set. I've added a temporary debug commit to hopefully gather some insights here.
(The test wasn't run in previous CI runs since those failed during the serial tests, skipping the parallel ones then. I've also rebased on `master` -- hope you don't mind; feel free to revert that force push.)

Reply:
`testing_config` does not activate environments by default; added a commit to fix that.

Reply:
Thanks